db/migrate/20171010075404_add_loction_to_restaurant_metadata.rb

```diff
@@ -0,0 +1,6 @@
+class AddLoctionToRestaurantMetadata < ActiveRecord::Migration[5.1]
+  def change
+    add_column :restaurant_reviews_metadata, :restaurant_location_lng, :float
```

Comment:
why is the location in the `restaurant_reviews_metadata` table and not `restaurants`? it's not related to the reviews